### PR TITLE
Fix minor typo in auth docs

### PR DESCRIPTION
**Pull request by nikhil-thomas**
Fix a minor typo in a `kubectl apply` command

Signed-off-by: Nikhil Thomas <nikhilthomas1@gmail.com>

# Changes

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [ ] [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [ ] [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Release notes block below has been filled in or deleted (only if no user facing changes)

# Release Notes

```release-note
NONE
```

### CODE REVIEW

**Review by pradeepitm12 — COMMENTED** [review]
/lgtm

File: docs/auth.md [hunk 1]
```diff
@@ -305,7 +305,7 @@ specified in the `Run`.
 1. Execute the `Run`:
 
    ```shell
-   kubectl apply --filename secret.yaml serviceaccount.yaml run.yaml
+   kubectl apply --filename secret.yaml,serviceaccount.yaml,run.yaml
```
Comment (vdemeester) [inline docs/auth.md:308]: I don't think that's valid either. `--filename` needs to be repeated

```bash
kubectl apply -f ./my1.yaml -f ./my2.yaml # short flags
```

**Review by vdemeester — CHANGES_REQUESTED** [review]

File: docs/auth.md (hunk 1 above)
Comment (nikhil-thomas) [reply docs/auth.md:308]: it worked when i tried

```
kc apply -f https://raw.githubusercontent.com/tektoncd/pipeline/main/config/200-serviceaccount.yaml,https://raw.githubusercontent.com/tektoncd/pipeline/main/config/200-role.yaml,https://raw.githubusercontent.com/tektoncd/pipeline/main/config/201-rolebinding.yaml
```

kubectl docs: 
```
filename      f      []     that contains the configuration to apply
```
https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#apply

**Review by vdemeester — APPROVED** [review]
/cc @afrittoli @pritidesai @bobcatfish

**Pull request merged**